packages/remix-server-runtime/entry.ts

```diff
@@ -28,7 +28,10 @@ export interface AssetsManifest {
   routes: RouteManifest<EntryRoute>;
   url: string;
   version: string;
-  hmrRuntime?: string;
+  hmr?: {
+    timestamp?: number;
+    runtime: string;
+  };
```

Comment:
@pcattori I'm pretty sure this is just a type bug fix - but could use confirmation on that and that this isn't by design for any reason